### PR TITLE
kafka topic waiter add not found checks 

### PR DESCRIPTION
**Pull request by ivan-savciuc**
## About this change—what it does

Add a not found checks: 100 to the Kafka Availability waiter

## Why this way

To avoid `Error: error waiting for Aiven Kafka topic to be ACTIVE: couldn't find resource (21 retries)` when running TF client with height parallelism.

### CODE REVIEW

**Comment by ivan-savciuc** [comment]
Hi @kkonstan-ovo, thanks for your fatback earlier. This PR should improve the situation. Would you mind checking it?

File: go.sum [hunk 1]
```diff
@@ -10,24 +10,13 @@ github.com/acomagu/bufpipe v1.0.3 h1:fxAGrHZTgQ9w5QqVItgzwj235/uYZYgbXitB+dLupOk
...
-github.com/aiven/aiven-go-client v1.7.1-0.20220711165623-353150bcf2db/go.mod h1:0MOgKonaIxMT8aXK7D6xbaKBdUTGfc0GdNRn9kf/yQ0=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720094957-dc824e1c9710 h1:KhFYrsf6jwLb6IPZWmz0i/86nfAG/2ueAWt16GzqbYw=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720094957-dc824e1c9710/go.mod h1:0MOgKonaIxMT8aXK7D6xbaKBdUTGfc0GdNRn9kf/yQ0=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720101020-be517bd23aab h1:o/1VDjqPmgvtsCVtuR5rBHCRilHOPdxjwumvrODl8HQ=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720101020-be517bd23aab/go.mod h1:0MOgKonaIxMT8aXK7D6xbaKBdUTGfc0GdNRn9kf/yQ0=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720101328-5045a2302ed0 h1:pzG0yAwcawkg+D+WdYX1as5jVpwbk7yGXAFt2R6xguM=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720101328-5045a2302ed0/go.mod h1:0MOgKonaIxMT8aXK7D6xbaKBdUTGfc0GdNRn9kf/yQ0=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720101838-3c052e7bbc87 h1:fi+zsf/lmh6RwGDCLJPR8ioADQSR1K8jiSV1nhim49E=
-github.com/aiven/aiven-go-client v1.7.1-0.20220720101838-3c052e7bbc87/go.mod h1:0MOgKonaIxMT8aXK7D6xbaKBdUTGfc0GdNRn9kf/yQ0=
```
Comment (Serpentiel) [inline go.sum:22]: I'm sorry, but what happened here?

**Review by Serpentiel — APPROVED** [review]
check the comment, otherwise lgtm

**Comment by Serpentiel** [comment]
@kkonstan-ovo could you please try and see if this branch improves it?

File: go.sum (hunk 1 above)
Comment (mhoffm-aiven) [reply go.sum:22]: go mod tidy probably

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**